### ISSUE TITLE
Turkish is not Italian

### ISSUE THREAD

**Issue by tracyjanick** [jezen/is-thirteen#581]
tap.equal(is("on üç").thirteen(), true); // Italian

This one is Turkish language ('on üç'), not Italian

**Comment by ckilcrease** [comment]
True; this should be an easy fix (comment typo) in the file [test.js](https://github.com/jezen/is-thirteen/blob/master/test.js)
